Clinical-Genomics/scout
# Redirect to anchor when updating ClinVar Casedata and improved submission howto

This PR:
- Makes the vertical scrollable div of the ClinVar submissions page a bit higher.
- Introduces redirect to the specific submission casedata table when a user modifies the name of submission cases (fix #2468)
- Improves submission instructions for users who edit exported files with a non-English Excel software (fix #2443):

![image](https://user-images.githubusercontent.com/28093618/112299155-d071c780-8c97-11eb-9ab1-c0d42efd6c3a.png)


**How to test**:
1.  Create more than 1 ClinVar submission (remember to pin variants to see the option).
2. Edit the names of the individuals in casedata of one of the submissions located at the bottom of the page and make sure that after saving the new names the page redirects to the same panel.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
